draft-ietf-tls-tls13.md

```diff
@@ -4319,6 +4322,31 @@ will receive various TLS 1.x versions in this field, however its value
 MUST always be ignored.
 
 
+## Zero-RTT backwards compatibility
```

Comment:
I considered that, but this section contains both client and server advice. Splitting them up is confusing because the reasoning is the same (and the server advice does not fall under "Negotiating with an older client"). I think it's best in its own section because this is a peculiar beast we have written ourselves into.